amplify/generated/models/AWSTweet.swift

```diff
@@ -2,7 +2,13 @@
 import Amplify
 import Foundation
 
-public struct AWSTweet: Model {
+public struct AWSTweet: Model,Hashable {
```

Comment:
do NOT edit this file. It is generated by AWS you should use an extension in a different file, you'll see there's a folder called "AmplifyModelExtensions"

There's already a file "AWSTweet+Extension" you should add your Hashable inheritance to that instead.